pages/dkp/kommander/2.2/workspaces/applications/platform-applications/application-deployment/index.md

```diff
@@ -68,7 +68,7 @@ Review the [list of available applications](../../platform-applications#workspac
       namespace: ${WORKSPACE_NAMESPACE}
     spec:
       appRef:
-        name: metallb-0.12.2
+        name: metallb-0.12.3
```

Comment:
since we are moving metallb deployment out of kommander (not yet removed as an app) into cluster provisioning step, we need to change out all of the metallb examples we are using (in quite a few places iirc)

@cmastr